frontend/src/routes/oauth.github.callback.tsx

```diff
@@ -20,6 +21,8 @@ function OAuthGitHubCallback() {
   React.useEffect(() => {
     if (isSuccess) {
       setGitHubToken(data.access_token);
+      // default analytics consent to true after successful login
+      handleCaptureConsent(true);
```

Comment:
Sorry, am I reading this right, why are we assuming consent?

Reply:
@amanape miscommunication here--this should only happen in SaaS mode since we've gotten ToS consent there. in OSS mode we should still do this